ietf-optical-impairment-topology.tree

```diff
@@ -49,39 +56,110 @@ module: ietf-optical-impairment-topology
                 +--:(concentratedloss)
                    +--ro concentratedloss
                       +--ro loss?   decimal64
-  augment /nw:networks/nw:network/nw:node/tet:te/tet:tunnel-termination-point:
+  augment /nw:networks/nw:network/nw:node/tet:te
+            /tet:tunnel-termination-point:
     +--ro OTSiG-element* [OTSiG-identifier]
     |  +--ro OTSiG-identifier    int16
     |  +--ro OTSiG-container
     |     +--ro OTSi* [OTSi-carrier-id]
     |        +--ro OTSi-carrier-id           int16
+    |        +--ro transponder-ref?          leafref
+    |        +--ro transceiver-ref?          leafref
+    |        +--ro configured-mode?          leafref
     |        +--ro OTSi-carrier-frequency?   decimal64
     |        +--ro OTSi-signal-width?        decimal64
     |        +--ro channel-delta-power?      decimal64
```

Comment:
Do we need this attribute?

Reply:
Delta-power defines the carrier power with respect the nominal-power that is the default power after roadm and after each amplifier. The meaning of this parameter in the transponder context is doubtful. I agree.
If we all agree we can delete from the list

Reply:
Delete the attribute